spec/services/admin/flags/notify_user_spec.rb

```diff
@@ -24,5 +24,11 @@
     it 'creates a notification' do
       expect { service }.to change { Notification.count }.from(0).to(1)
     end
+
+    it 'updates the flag\'s project submission to set a discard at date seven days in the future' do
+      freeze_time do
```

Comment:
When I initially ran it the test failed because it's comparing datetime and for some reason it seems to have a problem with the difference in milliseconds.

Let me have another look at it.

Reply:
> 
> 
> I've just had a play with this locally, it functions great! theres a few little UI things I think might need some tweaking:
> 
>     * We need a blank slate for when there are no notifications, at the moment it just displays a blank screen.
> 
>     * The is pushing up the page when there isn't enough notifications to fill the page, the footer should be at the bottom of the page.
> 
>     * I think we use a bell icon with a [red unread count like this](https://image.similarpng.com/very-thumbnail/2020/08/Youtube-notification-bells-Unread-Clipart-PNG.png) instead. I think it will be more familiar than the tilted icon with blue color.
> 
> 
> These are just a few little tweaks though, the main thing is how it functions and its spot on in that regard, nice work 👏

I didn't mess with the footer so it's always operated this way. We just never had a page without enough content. I'll have to have a proper look and probably bug Cody for some help.